test_crates/traits_with_types/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+#![feature(associated_type_defaults)]
```

Comment:
Can you check if the rustdoc generation script works on a stable Rust release with this feature enabled?

If not, we might need to get creative or, worst case, just not have a default associated type test case if we can't fit it in.

Reply:
Switching rustup to stable then regenerating the rustdocs results in no errors, so I don't think this is a problem

Reply:
Nice! Yes, I see now that we're generating the rustdoc with `RUSTC_BOOTSTRAP` which is a bit sketchy but in this case the right thing to do 😅 Thanks for verifying!